### PR TITLE
Adds dumb fuck juice

### PR DESCRIPTION
**Pull request by JF260**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: Added dumb fuck juice
/:cl:

## About The Pull Request
Dumb fuck juice is a new mixed drink. It's ingredients are one part Impedrezene, one part Syndicate Bomb, and one part Plasma.
      It has the effect of giving the drinker a random trauma; however, the chances of a special or magic trauma are much, much higher. This effect only occurs when you first metabolize the drink.
     The chances are 15% minor trauma, 15% major trauma, 20% magic trauma, 50% special trauma.

## Why It's Good For The Game
The main reason i implemented this is because there are two parts of the game that felt incomplete to me. The bartender has a ton of drinks, but their effects are all quaint and useless, while traumas have incredibly cool and unique effects, but have a 0.01% chance of ever being gotten outside of events. I figured the easiest way to complete both was to just mash them together.
     While the recipe seems a bit simple for such a powerful drink, it can be frustrating to have to rely on other departments for stuff in a server where each department is essentially an opposing tribe. I think we should test it for a bit before we make the recipe impossible.

(PS: The sprite is still vanilla pudding, mainly because I'm not well versed in making sprites and want to get this done and pull requested tonight. If it gets merged, I'll make a sprite asap.)

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
pretty interesting idea, I like more bartender drinks.

**Comment by enthethan** [comment]
this is just my honest nondesigner nonmaintainer regular player opinion: bartender and chef both need more recipes for their roles to be made more interesting and they especially need recipes which are useful (bit unfair that through chemistry you can get everything you could possibly want and bartenders just have couple of different niche recipes they can get)

File: hippiestation/code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -5,3 +5,12 @@
 	required_reagents = list(/datum/reagent/medicine/diphenhydramine = 1, /datum/reagent/medicine/morphine = 1, /datum/reagent/consumable/space_up = 1)
 	mix_message = "The ingredients come together to form a thick neon syrup."
 	mix_sound = 'hippiestation/sound/effects/lean_pour.ogg'
+	
+	
+/datum/chemical_reaction/lean
+	name = "Dumb Fuck Juice"
+	id = "dumbfuckjuice"
+	results = list(/datum/reagent/consumable/lean = 3)
```
Comment (ghost) [inline hippiestation/code/modules/food_and_drinks/recipes/drinks_recipes.dm]: forgot to change that to `/datum/reagent/consumable/dumbfuckjuice `

**Comment by karmaisblackandbluepilled** [comment]
+1

File: hippiestation/code/modules/food_and_drinks/recipes/drinks_recipes.dm (hunk 1 above)
Comment (JF260) [reply hippiestation/code/modules/food_and_drinks/recipes/drinks_recipes.dm]: Oops fuck, sorry

Comment (JF260) [reply hippiestation/code/modules/food_and_drinks/recipes/drinks_recipes.dm]: image

Comment (JF260) [reply hippiestation/code/modules/food_and_drinks/recipes/drinks_recipes.dm]: Fixed

**Comment by yoyobatty** [comment]
This is a web editor PR and I bet it hasn't been tested. The ingredients are somewhat fair and require a tad collaboration with bar to make so it isn't incredibly op. It can potentially be beneficially to the owner if they get a magic trauma but this can also massively fuck over if used on others. This is going to require some thorough input and explanations.

**Comment by SireTurret** [comment]
The issue I see with this is that it might be used to frick people by giving them a bunch of brain traumas. Other than that, I'd like if the bartender had more drinks. Chemistry already has a lot of content.

**Review by JF260 — COMMENTED** [review]
> 
> 
> The issue I see with this is that it might be used to frick people by giving them a bunch of brain traumas. Other than that, I'd like if the bartender had more drinks. Chemistry already has a lot of content.

I mean, if you wanted to do that, it'd be easier to just feed em impedrezene. Nonwilling injection of both is valid anyway.

**Comment by ghost** [comment]
Plus uno, more fun is more fun.

**Comment by AnthonyJNgo** [comment]
Interesting idea, flesh it out more and get the bugs ironed out and I could see this being an easy PR to merge.

**Comment by AnthonyJNgo** [comment]
I'm going to need to see a comment from you on this PR, a commit or something showing you are still working on this.

**Pull request closed without merging**